pkg/fixer/fixes/options.go

```diff
@@ -0,0 +1,12 @@
+package fixes
+
+import "github.com/open-policy-agent/opa/format"
+
+type Options struct {
+	Fmt *FmtOptions
+}
+
+type FmtOptions struct {
```

Comment:
FmtOptions was meant to be an options struct for all the known fixes but I think that `regal lint --fix` won't have any options. and the LSP will just call  fixes on their own so this feels wrong...

Reply:
If we used `regal fix` I guess the options could be to provide `--rules` similarly to the `lint` command. But yeah, I don't imagine we'll want configuration options for individual remediations.

Reply:
This has all gone now. no-whitespace-comment has been added too.

Violations will only be fixed where there is an instance in the lint report. So this `take existing ruleconfiguration into account` should happen by default.